pkg/operator/ceph/controller/spec.go

```diff
@@ -661,8 +663,23 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 	return sec
 }
 
-// LogCollectorContainer runs a cron job to rotate logs
+// LogCollectorContainer rotate logs
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+
+	maxLogSize := "500M"
+	if c.LogCollector.MaxLogSize != nil {
+		maxLogSize = fmt.Sprintf("%dM", c.LogCollector.MaxLogSize.Value()/1024/1024)
+	}
```

Comment:
actually, I went to a different approach,  instead of `1024` I'm, using `1000` for conversion. Because
1. `.Value()` from maxSize return in `bytes` so conversion should be on `1000` scale. `1 Byte= 0.000001 MB`
2. remove extra code to handle and return errors.

Look at the unit test for more examples

@BlaineEXE @travisn